Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -68,6 +69,11 @@ public MixedRealityToolkitConfigurationProfile ActiveProfile
         {
             get
             {
+                if (activeProfile == null)
+                {
+                    // NOTE: To avoid a circular reference, this message must not be controlled by the DebugUtilties class.
+                    Debug.LogError("Critical: The MixedRealityToolkit is not configured with a profile.");
```

Comment:
What are the valid cases?

Reply:
We weren't  checking here to begin with. So apparently there were valid cases before this change.

Reply:
I wonder if those aren't valid cases but weren't caught because they weren't checked, and we should find a way to fix those up now that it's properly checking.

Reply:
Pretty sure they're valid.